### ISSUE TITLE
Embeddings from the trained model don't work. 

### ISSUE THREAD

**Issue by tarunn2799** [huyvnphan/PyTorch_CIFAR10#14]
Hey, I loaded your pretrained model for CIFAR-10 using the code given in the readme, and then simply tried performing inference on my own with the model. For some reason, I'm getting very poor accuracy metrics (11%) on the CIFAR 10 test set. 

```
from torchvision.datasets import CIFAR10

dataset_dir = "/content/"
mean = [0.4914, 0.4822, 0.4465]
std = [0.2023, 0.1994, 0.2010]
transform_val = transforms.Compose([transforms.ToTensor(), transforms.Normalize(mean, std)])
dataset = CIFAR10(root= dataset_dir, train=False, transform=transform_val,  download = True)
dataloader = torch.utils.data.DataLoader(dataset, batch_size=128, num_workers=4, pin_memory=True)
```

Then I perform simple inference with `y=model(x)` format to get the embedding, and I argmax the 10-dim values I get to obtain my predicted class values. 

Also when I normalise, most of the values just turn out to be class 1 after I argmax. Very weird.

Am I doing something wrong here? I get 11% accuracy doing this.

**Comment by huyvnphan** [comment]
Have you put your model in eval mode? Please try again with the new version.
